rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/Mapping.java

```diff
@@ -30,9 +30,8 @@
   public static final String URL_VERSION = PRE + "version";
   public static final String URL_TASKS = PRE + "tasks";
   public static final String URL_TASKS_ID = URL_TASKS + "/{taskId}";
-  public static final String URL_TASK_GET_POST_COMMENTS = URL_TASKS_ID + "/comments";
-  public static final String URL_TASK_COMMENTS = URL_TASKS + "/comments";
-  public static final String URL_TASK_COMMENT = URL_TASK_COMMENTS + "/{taskCommentId}";
```

Comment:
The change of these Url's are breaking aswell, right?

Reply:
no. See above.

Reply:
If I would have changed anything, this should have an impact on the Controller, shouldn't it?